### DISCUSSION TITLE
No more showing new packages?-

### DISCUSSION THREAD

**Discussion by ocean** [Homebrew/discussions#4270] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.3-149-g66303bd
ORIGIN: https://github.com/Homebrew/brew
HEAD: 66303bd0926b391599d4c275ebc40928c8aed599
Last commit: 2 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: f7259742c77f0227a18d29c854b003d6bdcafa85
Core tap last commit: 4 weeks ago
Core tap branch: master
Core tap JSON: 26 Feb 00:13 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.2 => /opt/homebrew/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

When I run `brew update` and then `brew outdated`, there used to be a list of packages that were newly added to Homebrew show up, as well as the list of outdated packages I had already installed.

I don't see this list any more, has this feature been removed??

(I noticed this before the `4.0.0` release, not just recently)

**Comment by p-linnane** [comment, marked as answer]
With `4.0.0` we switched over to an API for updates/installs/etc. The API doesn't currently support showing the new packages.

There is an open PR to reimplement this: https://github.com/Homebrew/brew/pull/14802

**Reply by ocean** [reply]
Ah ok, thanks very much for the clear explanation @p-linnane 👍 

I'll take a look at the ongoing reimplementation.
